shell-completion/bash/journalctl

```diff
@@ -79,7 +68,7 @@ _journalctl() {
                                 comps='short short-iso short-precise short-monotonic verbose export json json-pretty json-sse cat'
                         ;;
                         --field|-F)
-                                comps=${__journal_fields[*]}
+                                comps=$(journalctl --fields | sort 2>/dev/null)
```

Comment:
Yeah, good catch. This code predated `--fields`, and should have been updated when `--fields` was added.